cmd/bump.go

```diff
@@ -209,11 +210,18 @@ var bumpCmd = &cobra.Command{
 			log.Fatal(err)
 		}
 
-		name, email := getGitAuthor()
+		name, email, err := getGitAuthor()
+
+		if err != nil {
+			fmt.Printf("git author name %s", name)
```

Comment:
log.debug

Reply:
Yeah I need to do that everywhere. Going to follow up and clean that up.